src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
+    } else {
+      Animal.alive.push(this);
+    }
```

Comment:
you will check this condition every time
but you can avoid this if set static `alive` property as empty array by default